captum/optim/_param/image/images.py

```diff
@@ -180,12 +208,31 @@ def torch_fftfreq(v: int, d: float = 1.0) -> torch.Tensor:
         return torch_rfft, torch_irfft, torch_fftfreq
 
     def forward(self) -> torch.Tensor:
+        """
+        Returns:
+            output (torch.tensor): A spatially recorrelated tensor.
+        """
+
         scaled_spectrum = self.fourier_coeffs * self.spectrum_scale
         output = self.torch_irfft(scaled_spectrum)
         return output.refine_names("B", "C", "H", "W")
 
 
 class PixelImage(ImageParameterization):
+    """
+    Parameterize a simple image tensor.
```

Comment:
nit: Do you mind adding more description here ? Parametrizes with what ?

Reply:
I tried to improve the description, let me know if it still needs to be improved!